### ISSUE TITLE
Remove multi-swap extension (OR modify documentation) since by now (almost) everything can be done with hx-select-oob

### ISSUE THREAD

**Issue by oliverhaas** [bigskysoftware/htmx-extensions#124]
See my comment here https://github.com/bigskysoftware/htmx/issues/1512#issuecomment-2557935715.

Since 1.0.2 the multi-swap extensions became redundant, since hx-swap-oob and hx-select-oob started supporting nested swaps (see https://github.com/bigskysoftware/htmx/blob/master/CHANGELOG.md#102---2020-12-12).

This means the documentation of multi-swap (which says oob swaps can't handle nested swaps), as well as the actual existence of the extension itself, is misleading. I certainly used multi swap when I did not need to.

If there is anything I'm missing, let me know. Otherwise I'm open to making a PR.

**Comment by oliverhaas** [comment]
I've actually found reasons what multi-swap can do that hx-select-oob can't:

Since there is not `Hx-Reselect-Oob` response header, one has to use mutli-swap if one wants that functionality with response headers.

Same goes for htmx.ajax: Since there is not `selectOobOverride` (yet?) or similar, multi-swap here has some niche where one might want to use it.

So I would probably reduce the scope of this issue to **modifying the docs** so it's reflecting the information given here.

**Comment by Telroshan** [comment]
I agree with you @oliverhaas , we realized this 3 years later, that OOB swap could totally handle nested elements, see https://github.com/bigskysoftware/htmx/pull/2119
I could have sworn I had changed the multiswap docs along back then to remove that note, but it seems it was all just a dream!

We can't remove the extension entirely as existing websites could depend on it, but we certainly should update the documentation to avoid confusion about what OOB can and cannot do

If you're interested, please feel free to open a documentation PR!

**Comment by oliverhaas** [comment]
#148
